src/sagemaker/tensorflow/README.rst

```diff
@@ -7,6 +7,17 @@ models on SageMaker Hosting.
 
 Supported versions of TensorFlow: ``1.4.1``, ``1.5.0``, ``1.6.0``, ``1.7.0``, ``1.8.0``, ``1.9.0``, ``1.10.0``, ``1.11.0``.
 
++----------------------------------------------------------------------------------------------------+
+| WARNING                                                                                            |
++====================================================================================================+
+| We are adding a new way of providing your TensorFlow training script with TensorFlow version 1.11. |
+| This new way gives the user script more flexibility. We are calling this new feature Script Mode.  |
+| In addition we are adding Python 3 support with Script Mode.                                       |
+| Make sure you refer to the correct section of this README when you prepare your script.            |
+| For clarification we will name the current training script configuration Legacy Mode.              |
+| Legacy Mode is going to be deprecated early 2019.                                                  |
```

Comment:
The plan is to deprecate framework/legacy mode with the launch with TensorFlow 2. I think it should be safe to say early 2019 since TensorFlow 2 is coming out this year.